File: app/src/main/java/life/plank/juna/zone/data/network/model/Stadium.java
package life.plank.juna.zone.data.network.model;

import lombok.Data;

@Data
public class Stadium {
    private Integer capacity;
    private String name;
    private Integer id;
    private Integer foreignId;
    private String surface;
    private String address;
    private String city;
    private String imagePath;
}

File: app/src/main/java/life/plank/juna/zone/data/model/football/Stadium.kt
package life.plank.juna.zone.data.model.football

import android.os.Parcelable
import androidx.room.Entity
import kotlinx.android.parcel.Parcelize

@Entity
@Parcelize
data class Stadium(
        var capacity: Int?,
        var name: String?,
        var id: Int?,
        var foreignId: Int?,
        var surface: String?,
        var address: String?,
        var city: String?,
        var imagePath: String?
) : Parcelable